net/scripts/gce-provider.sh

```diff
@@ -192,7 +192,7 @@ cloud_CreateInstances() {
 
   (
     set -x
-    gcloud beta compute instances create "${nodes[@]}" "${args[@]}"
+    gcloud beta compute instances create "${nodes[@]}" "${args[@]}" --min-cpu-platform "Intel Skylake"
```

Comment:
I ran out of bashfu to include `--min-cpu-platform` in:
```bash
cpuBootstrapLeaderMachineType="--machine-type n1-standard-16"
```
A space, really google?  Did we learn nothing from "C:\Program Files"?

Reply:
```

dumb() {
  declare -a words=($1)

  for word in "${words[@]//%20/ }"; do
    echo "$word"
  done

}

x='hi there foo --min-cpu-platform Intel%20Skylake'

dumb "$x"
```




```

Reply:
expect:
```
hi
there
foo
--min-cpu-platform
Intel Skylake
```

Reply:
Thanks, I adapted this escaping in my latest commit.